### PR TITLE
cp-70: Fixed issues with file uploading functionality

### PR DESCRIPTION
**Pull request by Nanaier**
Fixed the following issues:

- When the user tries to add a file > than 10MB:
 Now the code is 400.

Screenshot 2023-09-13 at 16 02 03

- When the user adds a file in any other format than .png or .jpg
 Now the code is 400.

Screenshot 2023-09-13 at 16 02 49

- Updated swagger to include id, url, and contentType and dates

Screenshot 2023-09-13 at 16 32 05

- Updated data returned from the backend on file entry creation

Screenshot 2023-09-13 at 16 32 27

### CODE REVIEW

File: backend/src/packages/files/file.service.ts [hunk 1]
```diff
@@ -62,15 +62,14 @@ class FileService implements Service {
...
-    await this.fileRepository.create(
+    const file = await this.fileRepository.create(
       FileEntity.initializeNew({
         url,
         contentType: payload.contentType,
       }),
     );
 
-    return { url: presignedUrl };
+    return { id: file.toObject().id, url, contentType: payload.contentType };
```
Comment (liza-veis) [inline backend/src/packages/files/file.service.ts]:
```suggestion
    return file.toObject()
```

**Review by liza-veis — CHANGES_REQUESTED** [review]

File: backend/src/libs/plugins/file-upload/file-upload.plugin.ts [hunk 2]
```diff
@@ -15,7 +15,9 @@ const fileUpload = fp<Options>((fastify, { extensions }, done) => {
 
   fastify.addHook(ControllerHook.PRE_VALIDATION, async (request) => {
     if (!request.isMultipart()) {
-      return;
+      throw new FileError({
```
Comment (dmytro-kucherenko) [inline backend/src/libs/plugins/file-upload/file-upload.plugin.ts]: You're throwing file error, that as I understand has internal server error status. Should this error description also be in swagger documentation?

Comment (Nanaier) [reply backend/src/libs/plugins/file-upload/file-upload.plugin.ts]: hey - I checked once again and catching this error breaks down the logic behind generating swagger documentation, so this check had to be removed

Comment (dmytro-kucherenko) [reply backend/src/libs/plugins/file-upload/file-upload.plugin.ts]: Nice, got it

**Review by dmytro-kucherenko — APPROVED** [review]

**Review by liza-veis — APPROVED** [review]

**Pull request merged**